bin/truffleruby.sh

```diff
@@ -82,6 +82,10 @@ fi
 java_args=()
 CP=""
 
+# Increase the Metaspace size to avoid a Full GC during startup,
+# triggerred by the default MetaspaceSize (~20MB).
```

Comment:
Typo: "triggerred" -> "triggered".

Reply:
My spell checker doesn't check .sh files for some reason.
Will fix.